oci-artificial-intelligence/ai-language/custom-model/custom-model.md

```diff
@@ -0,0 +1,381 @@
+
+4. **Model details**: In the "Model details" step, you will name your model, add a description of it and optionally add tags.
+
+    ![](./images/add-model-details.png " ")
+
+5. **Review and Submit**: In the "review" step, you can verify that all of your information is correct and go back if you want to make adjustments. When you want to start training, click "Create and train" and this will kick of the process. You can then check on the status of your model in the project where you created it.
+
+    ![](./images/ner-review.png " ")
+
+## **Task 3:** Create and Train Custom TXTC Model
+
+1. **Navigate to Models**: Under models, click on create and train model.
+
+2. **Choose model type**: Choose Text classification for model type. Select Single label or multi label as classification model type depending on your labeled data.
+
+    ![](./images/txtc-model-type.png " ")
+
+3. **Select training data**: Choose existing labeled data from Object Storate or Data Labeling service or you can choose to create new dataset by clicking Data science labeling link, which will drive you to OCI Data Labeling service, where you can easily add labels.
+
+     ![](./images/txtc-training-data.png " ")
+
+4. **Model details**: In the "Model details" step, you will name your model, add a description of it and optionally add tags.
+
+    ![](./images/add-model-details.png " ")
+
+5. **Review and Submit**: In the "review" step, you can verify that all of your information is correct and go back if you want to make adjustments. When you want to start training, click "Create and train" and this will kick of the process. You can then check on the status of your model in the project where you created it.
+
+    ![](./images/txtc-review.png " ")
+
+## **Task 4:** Create an Endpoint and Analyze through endpoint.
+
+1. Under Model endpoints, click on create model endpoint
+2. Specify the name of the endpoint (e.g., ticket_classification_endpoint1)
+    
+    ![](./images/create-model-endpoint.png " ")
+
+3. Once endpoint is active, Under Resources, click on Analyze link.
+4. Enter text, and click on Analyze to see the result
+
+    ![](./images/analyze.png " ")
+
+## **Task 5:** Create Custom Models and analyzing through endpoint with Python SDK.
+
+For setup, please refer Lab 1
+
+[Proceed to the next section](#next).
+Follow below steps to run Python SDK:
+
+#### 1. Download Python Code.
+
+#### Custom NER Python Code
+
+```Python
+<copy>
+import time
+import oci
+print(f'OCI Client SDK version: {oci.__version__}')
+config = oci.config.from_file()
+
+compartment_id = "ocid1.tenancy.oc1..aaaaaaaaih4krf4od5g2ym7pffbp6feof3rx64522aoxxvv3iuw3tam6fvea"  #TODO Provide your compartmentId here
+
+project_name = None #"custom-NER-project"
+model_name = None #"custom-NER-model"
+endpoint_name = None #"custom_NER_endpoint"
+
+#assuming ashburn endpoint.
+# For other regions, use below end points
+endpoint = "https://language.aiservice-preprod.us-ashburn-1.oci.oraclecloud.com"
```

Comment:
endpoint is not required, can be removed here and live 161
oci.ai_language.AIServiceLanguageClient

Reply:
Removed